config/default.yml

```
@@ -413,6 +413,11 @@ Rails/FindEach:
     - select
     - lock
 
+Rails/FreezeTime:
+  Description: 'Prefer `freeze_time` over `travel_to` with an argument of the current time.'
```

Can you add the style guide URL?

```suggestion
  Description: 'Prefer `freeze_time` over `travel_to` with an argument of the current time.'
  StyleGuide: 'https://rails.rubystyle.guide/#freeze-time'
```